webapi/src/main/scala/org/knora/webapi/messages/v2/responder/ontologymessages/KnoraApiV2WithValueObjects.scala

```diff
@@ -0,0 +1,2323 @@
+/*
+ * Copyright © 2015 Lukas Rosenthaler, Benjamin Geer, Ivan Subotic,
+ * Tobias Schweizer, André Kilchenmann, and Sepideh Alassi.
+ *
+ * This file is part of Knora.
+ *
+ * Knora is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Knora is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.knora.webapi.messages.v2.responder.ontologymessages
+
+import org.knora.webapi._
+
+/**
+  * Represents the `knora-api` ontology, version 2, in the [[ApiV2WithValueObjects]] schema.
+  */
+object KnoraApiV2WithValueObjects {
+    val Resource: ClassEntityInfoV2 = makeClass(
+        classIri = OntologyConstants.KnoraApiV2WithValueObjects.Resource,
+        cardinalities = Map(
+            OntologyConstants.SchemaOrg.Name -> Cardinality.MustHaveOne,
+            OntologyConstants.KnoraApiV2WithValueObjects.HasStandoffLinkTo -> Cardinality.MayHaveMany,
+            OntologyConstants.KnoraApiV2WithValueObjects.HasPermissions -> Cardinality.MustHaveOne,
+            OntologyConstants.KnoraApiV2WithValueObjects.CreationDate -> Cardinality.MustHaveOne,
+            OntologyConstants.KnoraApiV2WithValueObjects.LastModificationDate -> Cardinality.MayHaveOne
+        )
+    )
+
+    val IsEditable: PropertyEntityInfoV2 = makeProperty(
```

Comment:
same question as above: is it only present in JSONLD if set to true?

Reply:
Yes.

Reply:
It has to be this way, because I can't set it to false on properties we don't control, like `rdfs:label`.

Reply:
Could you explain what you mean by "we don't control"? Sufficient permissions provided, a resource's `rdfs:label` can be edited.

Or do you want to say that we can't make assertions about `rdfs:label` at all because it is defined elsewhere and not in our ontology?

Reply:
I mean the latter.